crates/typst/src/math/equation.rs

```diff
@@ -395,36 +410,82 @@ fn find_math_font(
     Ok(font)
 }
 
+/// A frame's size and insertion point relative to the parent frame.
+struct SizePoint {
```

Comment:
I don't feel like this struct adds much over a simple tuple. Is it really worth it?

Reply:
I think so.. when writing this function first with tuple instead of `SizePoint`, even I myself got confused by tuple's `.0` and `.1`, so I think using tuples would be more unreadable for others.

Reply:
resolving for now :)

Reply:
I would typically use restructuring instead of .0 and .1

Reply:
Makes sense, thx! Adopted in a new commit.